Automattic/wp-calypso
# Jetpack section: add Automated Transfer step for Business sites

#### Changes proposed in this Pull Request

* Show non-Atomic Business sites a CTA to Automated Transfer.

Fixes 1179060693083348-as-1179702911174270

Todo:
* Make the CTA do the AT.
* Make the CTA show a progress indicator as per design.

#### Testing instructions

**Prerequisite:** non-Atomic WPCOM site with Business plan.
* Run this PR and enable the `jetpack/features-section`.
* Visit Jetpack > Backup.
* Verify that you see the Automated Transfer CTA and that it matches the design.
* Visit Jetpack > Scan.
* Verify that you see the Automated Transfer CTA and that it matches the design.

#### Demo 
##### While trying to access the Jetpack Backup section
![image](https://user-images.githubusercontent.com/3418513/84550638-865bbe80-ace1-11ea-8380-6b9c70f8731b.png)

##### While trying to access the Jetpack Scan section
![image](https://user-images.githubusercontent.com/3418513/84550731-ccb11d80-ace1-11ea-938e-a66fdf2ffcd7.png)
